### ISSUE TITLE
Add storage group module to neofs-cli

### ISSUE THREAD

**Issue by realloc** [nspcc-dev/neofs-node#80]
Basic set of StorageGroup operations to implement:

- Create
- List StorageGroups for container
- Get StorageGroup by ID
  There should be an option to save SG to a file or print in a desired format
- Delete StorageGroup by ID

**Comment by alexvanin** [comment]
This module should create complete storage group objects. Storage group object payload contains list of members (objects to verify) and homomorphic hash of all member's payload combined.

To prepare this object CLI should:
1. for each passed object ID make `object.Head` request with `TTL>1 RAW=true main_only=true`,
2 if we got response then add it to member list, accumulate homomorphic hash and total size and go to **step 1**,
3. if we got split info error, then iterate through the chain of children by making same `object.Head` requests
4. add children in direct order to list of members, accumulate homomorphic hash and total size and go to **step 1**,
5. prepare storage group structure and finalized object with it,
6. make `object.Put` of finalized object.

Finalized object should be smaller than max object size of the network. If it is bigger, then remote node will return error on `object.Put` and this is correct expected behavior.
